files/zh-cn/web/javascript/reference/global_objects/arraybuffer/slice/index.md

```diff
@@ -1,42 +1,49 @@
 ---
 title: ArrayBuffer.prototype.slice()
 slug: Web/JavaScript/Reference/Global_Objects/ArrayBuffer/slice
+l10n:
+  sourceCommit: d254545601e07aa48a6bb9d5bfc1a75f6561de4d
```

Comment:
Please use the commit sha from [the corresponding file](https://github.com/mdn/content/commits/main/files/en-us/web/javascript/reference/global_objects/arraybuffer/slice/index.md)

```suggestion
  sourceCommit: 27180875516cc311342e74b596bfb589b7211e0c
```

Reply:
Got it.